### PR TITLE
BD-3246: Autosave for campaigns beta - MERGE 7/18

### PR DESCRIPTION
**Pull request by lydia-xie**
### Summary

Hidden doc for autosaving for campaigns

### Related PRs, issues, or features (optional)

- Closes [BD-3246](https://jira.braze.com/browse/BD-3246)

### Feature release date (optional)

- July 19, 2024

### Contributor checklist

- [ ] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

File: _docs/_hidden/private_betas/campaign_autosave.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+description: "This reference article covers details on how autosave works for campaigns."
+page_type: reference
+---
+
+# Autosaving campaigns
+
+> As you build your campaigns in Braze, your changes are now automatically saved. This way, you can focus on fine-tuning your campaign details with confidence, knowing that your progress is preserved.
+
+{% alert important %}
+Autosave is is currently in beta and only available for campaigns. Contact your customer success manager if you're interested in participating in this beta.
```
Comment (rachel-feinberg) [inline _docs/_hidden/private_betas/campaign_autosave.md:15]:
```suggestion
Autosave is currently in beta and only available for campaigns. Contact your customer success manager if you're interested in participating in this beta.
```

File: _docs/_hidden/private_betas/campaign_autosave.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## How it works
+
+![][1]{: style="float:right;max-width:40%;margin-left:15px;"}
+
+Your campaigns will automatically and periodically save as you edit and switch between tabs in the campaign editor.
+
+Changes are saved as a draft for both draft and active campaigns. For stopped campaigns, your changes will be saved, but the campaign will stay stopped.
+
+If you and another user make changes to a campaign, the first set of changes will be saved. If you're the second person to save changes, you'll need to refresh the page to see the latest updates to the campaign.
```
Comment (rachel-feinberg) [inline _docs/_hidden/private_betas/campaign_autosave.md:26]: Side note: I'm assuming there aren't indicators that two people are making changes at the same time?

Comment (lydia-xie) [reply _docs/_hidden/private_betas/campaign_autosave.md:26]: As far as I know, nope 🤔

**Review by rachel-feinberg — APPROVED** [review]
Lgtm!

**Pull request merged**